src/importexport/musicxml/internal/musicxml/exportxml.cpp

```diff
@@ -6775,6 +6775,19 @@ void ExportMusicXml::writeElement(EngravingItem* el, const Measure* m, staff_idx
     }
 }
 
+//---------------------------------------------------------
+//  clampMusicXmlOctave
+//---------------------------------------------------------
+
+/**
+ Clamps octave to min and max value as per MusicXML Schema
+ */
+
+static void clampMusicXmlOctave(int& octave)
+{
+    octave = std::clamp(octave, 0, 9);
```

Comment:
This seems to be C++17 and later only, am I right that `octave = std::max(std::min(octave, 9), 0);` is the same thing for older C++ standards?